### PR TITLE
Fixed MultiRoot problem

### PR DESCRIPTION
**Pull request by McRuben**
- [ ] breaking changes
- [x] no breaking changes

### changes made to the repository
Use the ServiceInfoSnapshots for every node and not only from the node where the process is located at

### CODE REVIEW

File: cloudnet/src/main/java/de/dytanic/cloudnet/CloudNet.java [hunk 1]
```diff
@@ -727,8 +727,10 @@ public void runCommand(ServiceInfoSnapshot serviceInfoSnapshot, String command)
 
     @Override
     public ServiceInfoSnapshot getCloudServiceByName(String name) {
-        ICloudService service = Iterables.first(cloudServiceManager.getCloudServices().values(), cloudService -> cloudService.getServiceId().getName().equalsIgnoreCase(name));
-        return service != null ? service.getServiceInfoSnapshot() : null;
+        return this.cloudServiceManager.getGlobalServiceInfoSnapshots().values().stream()
+                .filter(serviceInfoSnapshot -> serviceInfoSnapshot.getServiceId().getName().equalsIgnoreCase(name))
```
Comment (realPanamo) [inline cloudnet/src/main/java/de/dytanic/cloudnet/CloudNet.java]: Comparing the uuid of the service might be better here

Comment (McRuben) [reply cloudnet/src/main/java/de/dytanic/cloudnet/CloudNet.java]: What do you mean with that?

Comment (realPanamo) [reply cloudnet/src/main/java/de/dytanic/cloudnet/CloudNet.java]: Services have uuids, use the name to compare them

Comment (realPanamo) [reply cloudnet/src/main/java/de/dytanic/cloudnet/CloudNet.java]: Oh bruh

**Pull request merged**